src/core/sass.ts

```diff
@@ -145,6 +145,16 @@ export async function compileSass(
     '// quarto-scss-analysis-annotation { "origin": null }',
   ].join("\n\n");
 
+  const save_scss_prefix = Deno.env.get("QUARTO_SAVE_SCSS")
```

Comment:
Very minor, but we use camelCase throughout.

Reply:
Oops, fixed now!